templates/smart-start/document.html

```diff
@@ -9,5 +9,8 @@
         {{ document.body_html | safe }}
     </div>
 </div>
+<a href="/support/contact-us" class="js-invoke-modal">Get in touch&nbsp;›</a>
```

Comment:
Please use `&rsaquo;` instead of >

Reply:
This is just an example of this functionality. Galem is making a landing page, which I suppose is where I am supposed to add this link/button? I will hold off this PR until that is ready.